wolfcrypt/test/test.c

```diff
@@ -6797,9 +6790,6 @@ byte GetEntropy(ENTROPY_CMD cmd, byte* out)
 #ifdef FREESCALE_MQX
     #define CERT_PREFIX "a:\\"
     #define CERT_PATH_SEP "\\"
-#elif defined(WOLFSSL_MKD_SHELL)
```

Comment:
Just noticed the typo on this old line, should have been `WOLFSSL_MDK_SHELL`. Although it looks like this code is being removed?

Reply:
Thanks. It was looked over because of improved compatibility of MDK library. Prefix control is not needed for the library.